portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,9 +15,8 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Map.Entry;
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
Remove empty line. All imports are grouped together

Reply:
There was no empty line here...line 17 is the first import and line 18-22 are all imports.